prisma/schema.prisma

```diff
@@ -10,88 +10,82 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model User {
-  id        String   @id @default(cuid())
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
-
-  firstName            String?
-  lastName             String?
-  role                 Role      @default(USER)
-  email                String    @unique
-  phone                String?
-  stripeId             String?   @unique
-  hash                 String
-  allowMultiple        Boolean   @default(false)
-  resetPasswordToken   String?   @unique
-  resetPasswordExpires DateTime?
-
-  Shop        Shop?
-  Review      Review[]
-  UsersOnRoom UsersOnRoom[]
-
-  @@map("users")
+model Location {
+  id                Int     @id @default(autoincrement())
+  string            String?
+  frontage          String?
+  intersection      String?
+  intersection_type String?
+  dimensions        String?
+  hour              Int?
+  minute            Int?
+  distance          Int?
+  category          String?
+  gps_latitude      Float?
+  gps_longitude     Float?
+  Camp              Camp[]
+  Art               Art[]
 }
 
-model Shop {
+model Camp {
   id        String   @id @default(cuid())
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
 
-  name        String
-  description String?
-  website     String?
-
-  manager User?    @relation(fields: [userId], references: [id])
-  userId  String?  @unique()
-  reviews Review[]
-
-  @@index([name])
-  @@map("shops")
+  year            String
+  name            String
+  url             String?
+  contact_email   String?
+  hometown        String?
+  description     String?
+  location        Location? @relation(fields: [locationId], references: [id])
```

Comment:
I think i misunderstood how to add an object here, and instead added a whole new model. @thespacemaker ?

Reply:
Yea, that's the way SQL should work. You really only have primitives available as fields and more complex objects should be in their own table and related back with a relation and an ID.

https://www.postgresql.org/docs/8.1/datatype.html

for a list of all the available Postgres types. You do have the option of using JSON... but that really should be reserved for instances where you are saving data in which you do not know the structure of ahead of time.